packages/sync-actions/src/products.js

```diff
@@ -43,6 +43,7 @@ function createProductMapActions(
   ): Array<UpdateAction> {
     const allActions = []
     const { sameForAllAttributeNames } = options
+    const { publish } = newObj
```

Comment:
regarding this flag naming, i thought that `publish` is more clear than using `staged`.